src/js/Menus/DropdownMenu.d.ts

```diff
@@ -19,11 +19,27 @@ export interface DropdownMenuProps extends SharedDropdownMenuProps {
   children?: React.ReactElement<any>;
 }
 
-export interface DropdownMenuComponent extends React.ComponentClass<DropdownMenuProps> {
-  Positions: LayoverPositions;
-  HorizontalAnchors: HorizontalAnchors;
-  VerticalAnchors: VerticalAnchors;
+interface DropdownMenuComponent extends React.ComponentClass<DropdownMenuProps> {
```

Comment:
@dave-hillier Could you update the `DropdownMenuComponent` type to be exported as well? There was a PR a few patches ago to export all the component definitions. This PR looks good to me otherwise. Thanks!